kafkastats/src/main/java/com/pinterest/doctorkafka/stats/BrokerStatsRetriever.java

```diff
@@ -387,6 +387,19 @@ private void setBrokerInstanceInfo() {
         }
       }
     }
+    LOG.warn("set hostname to {}", brokerStats.getName());
```

Comment:
shall we only have LOG.warn if   brokerStats.getName() == null ?

Reply:
Yes, that only gets called if getName() returned null.  OTOH, that can totally be pulled out, as I only needed it to determine if my fix was working right.